test/unit/inet-address-tests.js

```diff
@@ -94,6 +94,19 @@ describe('InetAddress', function () {
       helper.assertInstanceOf(val, InetAddress);
       assert.strictEqual(val.toString(), '10.11.12.13');
     });
+    it('should parse IPv6 Addresses with Embedded IPv4 Addresses', function () {
+      [
+        ['0:0:0:0:0:FFFF:129.144.52.38',    '00000000000000000000ffff81903426'],
+        ['0:0:0:0:0:0:129.144.52.38',       '00000000000000000000000081903426'],
+        ['::ffff:129.144.52.38',            '00000000000000000000ffff81903426'],
+        ['::ffff:254.255.52.32',            '00000000000000000000fffffeff3420'],
+        ['::13.1.68.3',                     '0000000000000000000000000d014403'],
+        ['fe::13.1.68.3',                   '00fe000000000000000000000d014403']
```

Comment:
@jorgebay Sorry, didn't see the follow up messages but did see the merge. Weird.

Anyway, I would explicitly disallow the `::` case since it's been out of spec for a long time. Since this is new code and not supporting backward behavioral compatibility it seems like a good time to make "stable" choices like this.

If we were following the Java driver for guidance then we'd actually convert mapped addresses with both suffixes to IPv4 according to the (perceived?) original intent of the RFC. That seems even worse to me though because now I've written IPv6 and gotten IPv4 back.